### ISSUE TITLE
Delete {Variable} # Is not work!

### ISSUE THREAD

**Issue by CrazyWords1** [SkriptLang/Skript#1795]
### Description
Delete {Variable} is not work P.S (With all database | CSV | SQLite | MySql)

### Steps to Reproduce
```
Set {1} to 1
send "%{1}%"
delete {1}
send "%{1}%"
```

### Expected Behavior
```
1
<none>
But in "Release" 2.3
1
1
```

### Errors / Screenshots
none

### Server Information
* **Server version/platform:**      PaperSpigot 1.12.2 (Latest)
* **Skript version:**                2.3

**Comment by Snow-Pyon** [comment]
Can someone confirm this? I honestly doubt this is the case.

If it happened in a code with conditionals, make sure they're being met.

**Comment by TheBentoBox** [comment]
Uhhh...
![image](https://user-images.githubusercontent.com/6902229/50719504-f7b62d00-106a-11e9-9bd1-0d6832224ccb.png)
I'll note I DO have add-ons and other plugins and such, for what that's worth.

**Comment by Snow-Pyon** [comment]
Putting critical priority on this one since not being able to delete variables is a pretty critical thing. I'd be appreciated if someone could verify this without addons.

**Comment by TheBentoBox** [comment]
Confirmed without any add-ons or other plugins at all. 2.3, Paper 1.13.2.
![image](https://user-images.githubusercontent.com/6902229/50719624-2208ea00-106d-11e9-9845-33f1740ea226.png)

**Comment by Snow-Pyon** [comment]
Should be fixed in 2.3.1
